src/Network/Algorand/Client.hs

```diff
@@ -37,8 +37,8 @@ apiClient
 apiClient env = genericClientHoist $ \x ->
   liftIO $ runClientM x env >>= either throwM pure
 
-newtype AlgoNode = AlgoNode
-  { getAlgoNode  :: forall m . MonadIO m => NodeApi (AsClientT m)
+newtype AlgoNode m = AlgoNode
```

Comment:
hm.. why do you need this change?

Reply:
I assume that the `m` is the same with `forall m.` in `connectNode`, since when I work with this initially, there is an error about `m ~ m'`. But I guess this is unnecessary and revert it back.